src/CoreBundle/private/library/classes/dbobjects/TCMSPortal.class.php

```diff
@@ -192,8 +192,10 @@ public function IsDomainOfThisPortal($sDomainName)
     public function &GetLogo()
     {
         if (!array_key_exists('_portalLogo', $this->_aCache)) {
-            $this->_aCache['_portalLogo'] = new TCMSImage();
-            $this->_aCache['_portalLogo']->Load($this->sqlData['images']);
+            $portalLogo = new TCMSImage();
+            if(false !== $portalLogo->Load($this->sqlData['images'])){
```

Comment:
There are some spaces missing (start and end) -> cs fix

Reply:
thanks for this feedback. Re-formatted.